webui/react/src/e2e/fixtures/user.fixture.ts

```diff
@@ -14,6 +14,7 @@ interface UserArgs {
   username?: string;
   displayName?: string;
   isAdmin?: boolean;
+  password?: string;
```

Comment:
we want password in our `User` interface as well, right?

Reply:
maybe `EditArgs` can have optional password, and
```
interface UserArgs extends EditArgs
```
with password required?

Reply:
I don't think we want this in the `User` interface; I may be wrong, but based on how it's used it seems like it represents a server response, and the server wouldn't put the user password in a response.

Reply:
i'm just using it as an "internal test representation" of a user. think of it as "everything an admin would think of" when they think of a user. for example, if i'm gonna be logging in with that user later, i'm gonna need to know it's password.